cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/resource/KafkaRoller.java

```diff
@@ -193,7 +212,7 @@ public String toString() {
         singleExecutor.schedule(() -> {
             log.debug("Considering restart of pod {} after delay of {} {}", podId, delay, unit);
             try {
-                restartIfNecessary(podId, ctx.backOff.done());
+                restartIfNecessary(podId, ctx.backOff.done(), ctx);
```

Comment:
Do we need to pass both `ctx.backOff.done()` and `ctx`? Cant we pass just `ctx` and get `ctx.backOff.done()` from it later? Or does this have some meaning - e.g. in testing?